File: stecuanhueygarcia9.java
// Santiago Tecuanhuey Garcia
//
// Lab 9 - Java Version 

import java.util.Scanner;
import java.util.Random;

public class stecuanhueygarcia9{

	public static void main(String[] args){
	
	Scanner userInput  = new Scanner(System.in);
	int guess = 0, i, y; // test case with the variable x. Set x to 5 and 15.
			     
	Random randomInt = new Random();
	y = randomInt.nextInt(1,21);

	while(guess < 1 || guess > 20)
	{
	System.out.println("Enter a number between 1 and 20 :)");
	
	guess = userInput.nextInt();

	if (guess > 20 )
	{
	System.out.println("The number you entered is higher than my number, please try again!\n");
	}

	else if (guess < 1 )
	{
        System.out.println("The number you entered is lower than my number, please try again!\n");
        }
	
	}
if (guess == y)
{
	for(i = 0; i < guess; i++)
	{
	System.out.println("You won! \n");
	}
}

else if(guess > y)
{
	System.out.println("You lost :( The number you entered(" + guess + ") is bigger than my number " + y + ".");
}

else
{
	System.out.println("You lost :( The number you entered(" + guess + ") is lower than my number " + y + ".");
}
	
	}	
	
}

File: stecuanhueygarcia9.c
// Santiago Tecuanhuey Garcia 
//
//Lab 9 - C Version 
//


#include <stdio.h>
#include <stdlib.h>
#include <time.h>


int main(void){

int guess = 0, i, y; // test case with the variable x. Set x to 3, set x to 17.
srand(time(NULL));

y = 1 + (rand() % 21);

while(guess < 1 || guess > 20)
{	
	printf("Enter a number between 1 and 20 :) ");
	scanf("%d", &guess);

	if (guess > 20 )
	{
	puts("The number you entered is higher than my number, please try again!\n");
	}

	else if (guess < 1 )
	{
        puts("The number you entered is lower than my number, please try again!\n");
        }
}

if (guess == y)
{
for(i = 0; i < guess; i++)
	{	
	printf("You won! \n");
	}	
}

else if(guess > y)
{
printf("You lost :( The number you entered(%d) is bigger than my number(%d).\n", guess, y);
}

else{
printf("You lost :( The number you entered(%d) is lower than my number(%d).\n", guess, y);
}


return EXIT_SUCCESS;
}
